client/ayon_harmony/plugins/load/load_palette.py

```diff
@@ -8,66 +8,98 @@
 import ayon_harmony.api as harmony
 
 
-class ImportPaletteLoader(load.LoaderPlugin):
```

Comment:
By renaming this class name - any previously loaded container that uses `ImportPaletteLoader` can now NOT be managed anymore. Is that correct behavior? Were palettes already "unmanaged"? It doesn't seem so, since it did implement remove and update methods?

Reply:
It depends on what you call 'managed'? A loader was existing but was broken so I doubt anyone have built something on it.